src/ducks/components/App.jsx

```diff
@@ -31,10 +40,23 @@ export class App extends Component {
         <Main>
           <Switch>
             <Route path="/redirect" component={IntentRedirect} />
-            <Route path="/discover" component={Discover} />
-            <Route path="/myapps" component={MyApplications} />
-            <Redirect exact from="/" to="/discover" />
-            <Redirect from="*" to="/discover" />
+            {parts.map(name => {
+              if (componentsMap.hasOwnProperty(name)) {
```

Comment:
Is there a reason to use `hasOwnProperty` instead of just `componentsMap[name]`?

Reply:
A reflex, I removed it since we don't need it here 👍